package.json

```diff
@@ -11,7 +15,7 @@
     "why-is-node-running": "cli.js"
   },
   "engines": {
-    "node": ">=8"
+    "node": ">=20.11"
```

Comment:
I've raised the Node.js version requirements quite aggressively here due to the usage of [`import.meta.dirname`](https://nodejs.org/api/esm.html#importmetadirname), this can be lowered to 20 if need be, but would require a little bit of extra code.